src/components/SignupForm/SignupForm.js

```diff
@@ -1,22 +1,29 @@
-import { NavLink } from "react-router-dom";
+import { useContext } from "react";
+import { AuthContext } from "../../context/AuthContext";
 import "./SignupForm.css";
 
 export default function SignupForm() {
+  const { setUserName, setPassWord } = useContext(AuthContext);
+
   return (
     <div className="formWrapper">
       <form className="form">
```

Comment:
Did you test this form? Does it actually create new admins? 

I think we're missing an onsubmit function here 😅